hack/migrate-to-antrea.sh

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env bash
```

Comment:
I remember you said it will be done via a subcommand of antctl, any reason changed to a script?